docs/meshstack.index.md

```diff
@@ -36,6 +36,6 @@ The connections are shown in the following matrix table:
 | Cloud Foundry |              Organization               |                 Space                 |              UAA User              |                    Quota                    |
 |  Kubernetes   |                    -                    |               Namespace               |            Rolebinding             |               YAML Templates                |
 |   OpenShift   |                    -                    |                Project                |                User                |               YAML Templates                |
-|      AWS      |            Organization Unit            |                Account                |              IAM Role              |      CF StackSets / Organization Units      |
```

Comment:
d: perhaps we should somehow mention that for public cloud platforms the customer identifier is usually part of the tenant name (but it can actually be configured). Not sure whether that is already mentioned somewhere else. If not, i think it is an important information to add

Reply:
This is the overview page though. https://docs.meshcloud.io/docs/meshstack.index.html. I don't think that information fits here.